app/src/main/java/org/oppia/app/help/HelpFragment.kt

```diff
@@ -11,13 +11,15 @@ import javax.inject.Inject
 /** Fragment that contains help in the app. */
 class HelpFragment : InjectableFragment() {
   @Inject lateinit var helpFragmentPresenter: HelpFragmentPresenter
+  private var contextAttached : Context ?= null
```

Comment:
Remove this context and do not pass context to `HelpFragmentPresenter` instead you can use `activity` directly inside that file.